staff/durack/publication.html

```diff
@@ -51,17 +51,24 @@ <h2>Publications:</h2>
-<div data-badge-popover="right" data-badge-type="2" data-doi="10.5194/gmd-2020-426" data-hide-no-mentions="true" class="altmetric-embed"></div><br>
-Rieger, Landon A., Jason N. Cole, John C. Fyfe, Stephen Po-Chedley, Philip J. Cameron-Smith, <b>Paul J. Durack</b>, Nathan P. Gillett and Qi Tang (2020) Quantifying CanESM5 and EAMv1 sensitivities to volcanic forcing for the CMIP6 historical experiment, <i>Geoscientific Model Development</i>, <b>13</b> (10), pp 4831-4843. DOI: <a href="links/Riegeretal20GMD.html" target="_blank">10.5194/gmd-2019-381</a>
-<div data-badge-popover="right" data-badge-type="2" data-doi="10.5194/gmd-2019-381" data-hide-no-mentions="true" class="altmetric-embed"></div><br>
+<!--
+Tesdal, J-E., G. A. MacGilchrist R. L Beadling, S. M. Griffies, J. P. Krasting and <b>Paul J. Durack</b> (2022) The nonlinear impact of surface forcing changes on bottom water formation and overturning in the Southern Ocean, <i>Journal of Geophysical Research: Oceans</i>, <b>in press</b>, 45 pp6. DOI: <a href="links/Tesdaletal22JGR.html" target="_blank">10.1002/xxx</a>
+<div data-badge-popover="right" data-badge-type="2" data-doi="xx" data-hide-no-mentions="true" class="altmetric-embed"></div><br>
+Lambert, F. H., R. P. Allan, A. Behrangi, M. P. Byrne, P. Ceppi R. Chadwick, <b>Paul J. Durack</b>, G. Fosser, H. J. Fowler, P. Greve, T. Lee, H. Mutton, P. A. O'Gorman, J. M. Osborne, A. G. Pendergrass, J. T. Reager, P. Stier, A. L. S. Swann, A. Todd, S. M. Vincente-Serrano and G. L. Stephens (2022) Future changes in the regional water cycle and their impact on societies, <i>Nature Climate Change</i>, <b>in press</b>, 38 pp. DOI: <a href="links/Lambertetal22NATClimChg.html" target="_blank">10.1038/xxx</a>
+<div data-badge-popover="right" data-badge-type="2" data-doi="xx" data-hide-no-mentions="true" class="altmetric-embed"></div><br>
+Roethig et al
+-->
+McDougall, T. J., P. M. Barker, R. M. Holmes, R. Pawlowicz, S. M. Griffies and <b>Paul J. Durack</b> (2021) The interpretation of temperature and salinity variables in numerical ocean model output, and the calculation of heat fluxes and heat content, <i>Geoscientific Model Development</i>, <b>14</b> (10), pp 6445-6466. DOI: <a href="links/McDougalletal21GMD.html" target="_blank">10.5194/gmd-14-6445-2021</a>
+<div data-badge-popover="right" data-badge-type="2" data-doi="10.5194/gmd-14-6445-2021" data-hide-no-mentions="true" class="altmetric-embed"></div><br>
+Rieger, Landon A., Jason N. Cole, John C. Fyfe, Stephen Po-Chedley, Philip J. Cameron-Smith, <b>Paul J. Durack</b>, Nathan P. Gillett and Qi Tang (2020) Quantifying CanESM5 and EAMv1 sensitivities to volcanic forcing for the CMIP6 historical experiment, <i>Geoscientific Model Development</i>, <b>13</b> (10), pp 4831-4843. DOI: <a href="links/Riegeretal20GMD.html" target="_blank">10.5194/gmd-13-4831-2020</a>
```

Comment:
Should we add the "Mt. Pinatubo" to the title?
```suggestion
Rieger, Landon A., Jason N. Cole, John C. Fyfe, Stephen Po-Chedley, Philip J. Cameron-Smith, <b>Paul J. Durack</b>, Nathan P. Gillett and Qi Tang (2020) Quantifying CanESM5 and EAMv1 sensitivities to Mt. Pinatubo volcanic forcing for the CMIP6 historical experiment, <i>Geoscientific Model Development</i>, <b>13</b> (10), pp 4831-4843. DOI: <a href="links/Riegeretal20GMD.html" target="_blank">10.5194/gmd-13-4831-2020</a>
```